src/static/js/modules/nemo.js

```diff
@@ -130,7 +132,7 @@ $(document).ready(function (){
                     else {
                         thanks = "Thanks, weâ€™ll be in touch.";
```

Comment:
I know this was already existing but does the "we'll" format ok? Looks like it's encoded oddly.